data/card_catalogue/unique-1.json

```diff
@@ -514,7 +334,7 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Your next melee attack deals an additional d6 and sets your target on fire, or turns a slice of bread into toast. Your choice.",
+      "rules": "Your next melee attack deals an additional d6 and sets your target on fire.",
```

Comment:
I cut out the joke here since it doesn't have any context without the pidgeon game